### PR TITLE
FluentUI Charting Contrib Docsite using Docusaurus

### PR DESCRIPTION
**Pull request by Shubhabrata08**
FluentUI Charting Contrib Docsite using Docusaurus containing all markdown docs related to FluentUI React Charting along with gh-pages deployment workflow config files.
image
Here's a demo deployment:
https://shubhabrata08.github.io/fluentui-charting-contrib/

Note:
New MD docs are to be placed under docs folder in the root of the repo. The MD assets like images are to be placed in the corresponding assets folder under docs.
While adding images to markdown, please refer this doc in order to prevent broken links in deployment.
https://docusaurus.io/docs/static-assets

### CODE REVIEW

**Comment by AtishayMsft** [comment]
Can we keep the docs folder outside the docsite folder.

**Comment by Shubhabrata08** [comment]
We can configure the paths in the config file but might need to test if it works in build and deployment

**Comment by AtishayMsft** [comment]
Move images section under docs to /assets/images

**Comment by Shubhabrata08** [comment]
Those images are actually served as static files from the static directory, we might need to configure that as well...

**Comment by AtishayMsft** [comment]
> We can configure the paths in the config file but might need to test if it works in build and deployment

Lets do it. We can connect to decide the structure I have in mind.

File: apps/docsite/docusaurus.config.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+              to: "/docs/Fluent-React-Charting",
+            },
+          ],
+        },
+        {
+          title: "FluentUI",
+          items: [
+            {
+              label: "Public Developer Docsite",
+              href: "https://developer.microsoft.com/en-us/fluentui#/",
```
Comment (AtishayMsft) [inline apps/docsite/docusaurus.config.ts:108]: Update the href to https://aka.ms/fluentcharting

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**